src/styles/global.css

```diff
@@ -0,0 +1,34 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+@layer utilities {
+  .text-balance {
+    text-wrap: balance;
+  }
+}
+
+@font-face {
+  font-family: 'Lexend';
+  font-style: normal;
+  font-weight: 100 900;
+  font-display: optional;
+  src: url(/fonts/Lexend.ttf) format('woff2');
+}
+
+.MuiBox-root:nth-of-type(odd) {
+  background: rgb(217, 232, 245);
+}
+.MuiBox-root:nth-of-type(even) {
+  background: white;
+}
+
+
+main > article {
+  &:nth-of-type(odd){
+    background: rgb(217, 232, 245) !important;
```

Comment:
Why !important? Try to avoid !important as much as possible, since it breaks the natural flow of the cascading of the stylesheet.

Reply:
Not my code but will see if another solution is possible